examples/mujoco/mujoco_ppo_hl_multi.py

```diff
@@ -13,7 +13,6 @@
 """
```

Comment:
This script should be deleted as part of this PR, right?

Reply:
I thought I'd leave it in there until we decide on a final version. Then we'll have to change all examples to be consistent and make the new functionality visible

Reply:
Ok, makes sense. We can remove it once the new functionality is out of "alpha"